docs/bestpractices.md

```diff
@@ -103,4 +103,125 @@ If the changes are required to be applied immedietly, the previous inventory job
 ```
 microk8s kubectl delete job/snmp-splunk-connect-for-snmp-inventory -n sc4snmp
 ```
-The upgrade command can be executed again. 
\ No newline at end of file
+The upgrade command can be executed again. 
+
+
+### Traps messages don't appear in Splunk
+
+The first possible answer to why traps aren't sent to Splunk is that SNMP agents send trap messages to the wrong IP 
```

Comment:
I would add here a subtitle with as to any other problem that's _Wrong IP or port_ . Also let's not use short forms in documentation like _aren't_ etc.